File: C#/IsAnagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        
        if (s.Length != t.Length)
        {
            return false;
        }
        
        char[] sChar = s.ToCharArray();
        char[] tChar = t.ToCharArray();
        
        Array.Sort(sChar);
        Array.Sort(tChar);
        
        Console.WriteLine(sChar);
        Console.WriteLine(tChar);
        
        return (sChar.SequenceEqual(tChar));
    }
}

File: JavaScript/isAnagram.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isAnagram = function(s, t) {
    
    // Default
    if (s.length != t.length)
    {
        return false;
    }
    
    const sChar = s.split("");
    const tChar = t.split("");
    
    sChar.sort();
    tChar.sort();
    
    for (let i = 0; i < sChar.length; i++)
    {
        if (sChar[i] != tChar[i]) return false;
    }
    
    return true;
};